### DISCUSSION TITLE
LiveServer preview looks different than prod, Auto-CSS?

### DISCUSSION THREAD

**Discussion by turret001** [yandeu/five-server#101] [Help / Questions] [answered]
Hi! This is how my website looks on LiveServer:
![image](https://user-images.githubusercontent.com/24281147/229311916-a8a7b402-7207-4229-979a-4ccc31f2cf3f.png)

and this is how it looks when I open index.html on browser manually:
![image](https://user-images.githubusercontent.com/24281147/229311933-c75a1808-d399-4954-b47a-e4ddf7a209a1.png)

What could be causing this?
Dev tools screenshots that might be helpful:
LiveServer
![image](https://user-images.githubusercontent.com/24281147/229312013-8404a9fe-3fd0-4651-b4f8-288bf41b2a45.png)

Local
![image](https://user-images.githubusercontent.com/24281147/229312051-ac0961a2-3bf4-4a65-97f2-29f741ce42f7.png)

(I've also tried using chrome without extensions, issue persists)

**Comment by yandeu** [comment, marked as answer]
Isn't it just the zoom of the website? Try to adjust the zoom of the browser.
https://support.google.com/chrome/answer/96810

**Reply by turret001** [reply]
I'm speechless, how did I miss it.. lol

thanks for pointing out
